app/models/invoice.rb

```diff
@@ -8,4 +8,20 @@ class Invoice < ApplicationRecord
   has_many :transactions
   has_many :invoice_items
   has_many :items, through: :invoice_items
+
+  def find_invoice_item(invoice, item)
+    InvoiceItem.where(invoice: invoice, item: item)
```

Comment:
using .find_by instead of .where gives the object and not an array which I think will be better when it gets to the view.

Reply:
I changed this to use a find_by